contributing/developing.md

```diff
@@ -215,3 +220,29 @@ The npmrc allows for project-level [configuration](https://docs.npmjs.com/cli/v8
 ### .nvmrc
 
 Our nvmrc specifies the minimum node version required to run the project.
+
+### Development Mode Local Telemetry Logging
+
+During development and quality assurance testing, it can be helpful to validate telemetry events by logging to a local file when running the extension in Development Mode. Enable local dev mode telemetry logging using an advanced setting in your settings.json file:
+
+> "salesforcedx-vscode-core.advanced": {
+> "localTelemetryLogging": "true"
+> }
+
+With the above configuration, all extensions that re-use the core extension's telemetry module will log telemetry events to a local file at the project root.
+
+Extensions that are dependent on the core extension and use its telemetry module can also take advantage of this local logging functionality. For instance, the Einstein for Developers extension uses core telemetry but initializes its own instance. So with that extension installed the following line in settings.json will log telemetry events to a local file for Einstein for Developers only:
```

Comment:
I feel like this might be better to put in the E4D extension, since we don't do as much looking for docs over here. What do you think?